pyproject.toml

```diff
@@ -1,3 +1,77 @@
 [build-system]
-requires = ["setuptools >= 40.8.0", "wheel"]
+requires = ["setuptools>=61.0.0", "wheel"]
```

Comment:
Is `wheel` a required build dependency?

Reply:
iirc yes, because otherwise you only have source distributions instead of source (.tar.gz) and binary (.whl)

Reply:
I guess it depends on which build frontend we are using, with `build` and no `wheel` requirement I do get a binary distribution

Reply:
works for me, if you can lock/prove this behaviour through Github Actions then that's good enough for me

Reply:
I'm switching to `build` in CI